tests/data/nvidia_ai_gpu_chip_test.yaml

```diff
@@ -0,0 +1,30 @@
+scope:
+  fu:
+    description: "nvidia_ai_gpu_chip processing continuously over 5 years"
```

Comment:
that's new, what's the purpose?

Reply:
isn't the functional unit needing a description? the uf is usually covering a usage over a certain duration or distance or quantity...

Reply:
i removed it, it might be a duplicate of the report description